test/extended/cli/admin.go

```diff
@@ -528,6 +529,13 @@ var _ = g.Describe("[sig-cli] oc adm", func() {
 		oc.Run("delete").Args("-f", stableBusyboxPath).Execute()
 	})
 
+	g.It("release extract image-references", func() {
+		expected := string(testdata.MustAsset("test/extended/testdata/cmd/test/cmd/testdata/test-release-image-references.json"))
```

Comment:
Use `test/extended/testdata/` directory, in which case you'll just use `exutil.FixturePath("testdata", "test-release-image-references.json"")